removing some macros

DenisYaroshevskiy opened the pull request:
Summary: Will be making some changes in the surrounding code, they are going to be easier without macros

Differential Revision: D54001770

swolchok commented:
I am not confident that this didn't change generated code. Would like to see inspection of it in the test plan.

DenisYaroshevskiy commented:
Found a mistake in how I did the disassembly. I suggest to get agreement on the second diff (that I can only post on the internal, because it depends on this). If that's accepted, dissamble difference here becomes irrelevant.

DenisYaroshevskiy committed:
removing some macros (#120314)

Summary:

Will be making some changes in the surrounding code, they are going to be easier without macros

Test Plan:
`buck2 run fbcode//caffe2/test/cpp/jit:jit`

Asm comparison was requestied but proved difficult.

I suggest get agreement on the second diff and then it becomes irrelevant

Reviewed By: zhxchen17

Differential Revision: D54001770

DenisYaroshevskiy commented:
@pytorchbot merge
